src/equations/endurance.js

```diff
@@ -32,9 +32,12 @@ export default class Config {
     label: "Years Before Broke",
     type: "text"
   }
-  static equation = ({principal, burnAmount, totalInterest, deposits} = {}) => {
+  static equation = ({principal, burnAmount, burnPeriod, totalInterest, deposits} = {}) => {
```

Comment:
i wanted to see the select actually modify the result. i noticed that this equation isn't even using all of its params. is that intentional?

Reply:
endurance is an interesting equation if you work in all these args, but I really added it as a bullshit equation to get us up and running.  It's not entirely implemented.  Compounding these numbers wasn't the easiest thing to do.  We could even delete endurance if we want.